tests/containers/nfs/Containerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.fedoraproject.org/fedora-minimal:41
+
+RUN microdnf -y install /usr/bin/ps nfs-utils && microdnf clean all && rm -rf /var/cache/yum
```

Comment:
IIUC dnf5 obsoletes the need for microdnf?

Reply:
You're right ! Updated.